radish34/__tests__/integration.test.js

```diff
@@ -43,6 +51,22 @@ afterAll(async () => {
   await nativeClient.close();
 });
 
+describe('Get organization settings from config files', () => {
```

Comment:
Are you testing the test setup here? In the test? So meta... but might not be necessary 😛

Reply:
Haha yes I see your point. I was thinking if one of these tests fails it would indicate to the user that something had gone wrong prior to the testing. Most likely, the deploy process failed to properly create these files. I can delete these tests though and just simply import the config files if that's preferred

Reply:
Maybe this or some similar checks should go in the deploy app itself? Like, a post-deploy check? I know I’ve been bitten by this no-config-files bug.